### PR TITLE
Reliability cybertotal indicator

### PR DESCRIPTION
**Pull request by JasBeilin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/29347

## Description
Added reliability to cybertotal pack. PR will fail as the new parameter in YML file is a must. Should be further discussed in the demo.

## Screenshots
Reliability can be seen in the indicator's layout:
image

It cannot be seen while at indicator's screen. Is this working as designed? Aggregated should not take the only one if only one source reliability?

image

## Minimum version of Demisto
- [x] 5.0.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details: The new integrationReliability parameter is a must. 

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by Archibald7 — APPROVED** [review]

File: Packs/CyberTotal/ReleaseNotes/1_0_4.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CyberTotal
+- Added reliability parameter
```
Comment (roysagi) [inline Packs/CyberTotal/ReleaseNotes/1_0_4.md:4]:
```suggestion
- Added support for the *Source Reliability* integration parameter.
```

File: Packs/Base/ReleaseNotes/1_7_19.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CommonServerPython
+- Internal code improvements
```
Comment (roysagi) [inline Packs/Base/ReleaseNotes/1_7_19.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

**Review by roysagi — CHANGES_REQUESTED** [review]

File: Packs/CyberTotal/Integrations/CyberTotal/CyberTotal_test.py [hunk 3]
```diff
@@ -40,7 +40,7 @@ def test_ip(requests_mock):
         'threshold': 10,
     }
 
-    response = ip_reputation_command(client, args, 10)[0]
+    response = ip_reputation_command(client, args, 10, reliability='C - Fairly reliable')[0]
```
Comment (bakatzir) [inline Packs/CyberTotal/Integrations/CyberTotal/CyberTotal_test.py:43]: please consider validating the `DBotScoreReliability` object content

**Comment by bakatzir** [comment]
Can we close this PR? @JasBeilin

**Pull request closed without merging**